### ISSUE TITLE
[Bug]: MP4 files don't seem to save

### ISSUE THREAD

**Issue by LukeSaward1** [Eddio0141/UniTAS#161]
### What happened?

An MP4 file isn't created despite `movie.start_capture()` being in the movie.

### What should have happened?

The MP4 file should be created.

### UniTAS version

Latest master

### UniTAS setup tool

None

### BepInEx version

v5.3

### Game name

Do It For Me

### Game build

Windows x64 - Mono

### How to reproduce

Replay this movie file and it shouldn't generate an MP4 file.

### Log output

[LogOutput.log](https://github.com/Eddio0141/UniTAS/files/11012892/LogOutput.log)

### Additional context

Movie:
[movie.zip](https://github.com/Eddio0141/UniTAS/files/11012907/movie.zip)

**Comment by LukeSaward1** [comment]
Wait, FFMPEG doesn't seem to be found...

**Comment by Eddio0141** [comment]
Yeah I guess I should state in the movie logger that ffmpeg is missing to avoid confusion

Also the output mp4 file size is 1KB?

**Comment by LukeSaward1** [comment]
Yep, output size is 1KB. Not sure why.

**Comment by LukeSaward1** [comment]
Still happens on the latest commit (https://github.com/Eddio0141/UniTAS/commit/4eb63aed4b52a6990f0c0a015152f7cb5cd732ca).

**Comment by Eddio0141** [comment]
lmao see how it goes

**Comment by LukeSaward1** [comment]
Works fine now :D

**Comment by Eddio0141** [comment]
nice! glad its fixed
